src/Core/Worker.php

```diff
@@ -850,7 +850,7 @@ private static function nextPriority()
 		}
 
 		if (!empty($waiting)) {
-			$priority =  array_shift(array_keys($waiting));
+			$priority = array_keys($waiting)[0];
```

Comment:
Is it possible that index 0 doesn't exist?

Reply:
No, we first check that `$waiting` contains at least one element, which means `array_keys` will always return a list of at least one key.